### ISSUE TITLE
Unable to build servo demo with vendored source

### ISSUE THREAD

**Issue by happybeing** [tauri-apps/wry#1194]
**Describe the bug**
I need to build offline so am using `cargo vendor` in the wry repo on branch servo-wry-demo. When building it gets a long way, but at the cssparser it finds several unresolved imports:
```
   Compiling cssparser v0.31.2
error[E0432]: unresolved import `quote`
 --> /home/user/src/safe-servo/wry/../vendor-wry/cssparser/build/match_byte.rs:5:5
  |
5 | use quote::{quote, ToTokens};
  |     ^^^^^ use of undeclared crate or module `quote`

error[E0432]: unresolved import `syn`
 --> /home/user/src/safe-servo/wry/../vendor-wry/cssparser/build/match_byte.rs:9:5
  |
9 | use syn;
  |     ^^^ no external crate `syn`
[snip]
```

I guess this may be to do with servo rather than wry but starting here as I'm actually building the wry servo demo.

**Steps To Reproduce**
Though probably not relevant, I include the full steps I'm undertaking which start on a cloud server because my mobile broadband connection isn't able to do git remote operations on these large repos. After 'vendoring' on the cloud server I 'rsync' to my laptop and proceed from there.

On cloud server:
cd ~/servo
git clone https://github.com/tauri-apps/wry 
cd wry
git switch servo-wry-demo
cargo vendor ../vendor-wry/ > ../wry-cargo.config.toml

On Laptop
`#` Download mozjs binary and set the environment variable:
export MOZJS_MIRROR=/home/user/safe-servo/libmozjs-x86_64-unknown-linux-gnu.tar.gz

`#` Get the repos to laptop
cd ~/src && rsync --progress  -lHpErz user@cloudserver:/home/user/servo/ safe-servo/

`#` Add the following to ~/src/safe-servo/wry-cargo.config.toml
[net]
retry = 3                   # network retries
git-fetch-with-cli = true   # use the `git` executable for git operations
offline = true              # do not access the network

`#` Add contents of ~/src/safe-servo/wry-cargo.config.toml to ~/src/safe-servo/wry/.cargo/config.toml
`#` The above step gives cargo access to the vendored sources and prevents it from going online (which will fail for large repos).

`#` Then run the demo
cd ~/src/safe-servo/wry
cargo run --example servo

**Expected behavior**
Compilation succeeds and the servo demo runs.

**Platform and Versions (please complete the following information):**
OS: Ubuntu 22.04.04 LTS
cargo 1.74.1 (ecb9851af 2023-10-18)
rustc 1.74.1 (a28077b28 2023-12-04)

**Additional context**
I'd like to evaluate servo for use as a browser for a new p2p platform and have used Tauri previously, so wry with its servo demo looks like a promising place to start and put together an initial version of the browser.

**Comment by wusyong** [comment]
The demo has moved into an individual repo: https://github.com/european-browser/verso
Could you test if that's working for you?

**Comment by happybeing** [comment]
Thanks. I've tried the same approach (vendoring verso) and again a lot does compile. When it is trying to build servo it fails and I found workarounds for a couple of those issues but think that the way servo is built may not be compatible with this approach.

I'll close this now and look some more but it may have to wait until servo embedding is an option.

Thanks for your help. If you have rights to edit wry, maybe put the verso link in the servo-wry-demo README.md.

EDIT: I've opened an issue for verso explaining how I got around some issues trying to build vendored sources, but I haven't solved them all: https://github.com/european-browser/verso/issues/17
